# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/libreoffice/tarball-recompress.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- libreoffice-7.5.3.2/solenv/gbuild/UnpackedTarball.mk.vanilla	2023-05-22 09:58:15.395910032 +0000
+++ libreoffice-7.5.3.2/solenv/gbuild/UnpackedTarball.mk	2023-05-22 10:04:07.729905778 +0000
@@ -21,7 +21,6 @@
 define gb_UnpackedTarget__command_untar
 $(GNUTAR) \
 	-x \
-	$(3) \
 	-C $(UNPACKED_DIR) \
 	$(STRIP_COMPONENTS)=$(UNPACKED_STRIP_COMPONENTS) \
 	-f $(UNPACKED_TARBALL)
--- libreoffice-7.5.3.2/external/liblangtag/UnpackedTarball_liblangtag.mk.vanilla	2023-05-22 10:44:20.633876645 +0000
+++ libreoffice-7.5.3.2/external/liblangtag/UnpackedTarball_liblangtag.mk	2023-05-22 10:44:27.956876556 +0000
@@ -12,7 +12,7 @@
 $(eval $(call gb_UnpackedTarball_set_tarball,liblangtag,$(LIBLANGTAG_TARBALL),,liblangtag))
 
 $(eval $(call gb_UnpackedTarball_set_pre_action,liblangtag,\
-	$(GNUTAR) -x -j -f $(gb_UnpackedTarget_TARFILE_LOCATION)/$(LANGTAGREG_TARBALL) \
+	$(GNUTAR) -x -f $(gb_UnpackedTarget_TARFILE_LOCATION)/$(LANGTAGREG_TARBALL) \
 ))
 
 $(eval $(call gb_UnpackedTarball_update_autoconf_configs,liblangtag))
